### PR TITLE
Fixed a minor dependency consistency issue

### PR DESCRIPTION
**Pull request by davidfischer**
I basically just pinned a few dev dependencies and re-ran `./requirements/update.sh`

### CODE REVIEW

File: requirements/analyzer.txt [hunk 1]
```diff
@@ -122,6 +123,37 @@ numpy==1.26.4
...
+    # via torch
+nvidia-cuda-runtime-cu12==12.1.105
+    # via torch
+nvidia-cudnn-cu12==8.9.2.26
+    # via torch
+nvidia-cufft-cu12==11.0.2.54
+    # via torch
+nvidia-curand-cu12==10.3.2.106
+    # via torch
+nvidia-cusolver-cu12==11.4.5.107
```
Comment (ericholscher) [inline requirements/analyzer.txt:143]: I wonder why you got a bunch of cuda dependencies that I didn't 🤔 Perhaps different in OS or something?

**Review by ericholscher — APPROVED** [review]
This looks good, though I'm a little worried the cuda deps could baloon our install time even more.. not sure where they came from?

File: requirements/analyzer.txt (hunk 1 above)
Comment (davidfischer) [reply requirements/analyzer.txt:143]: I'm not sure exactly. Definitely odd.

Comment (davidfischer) [reply requirements/analyzer.txt:143]: I did run this again while fixing up #895 and got the same things.

**Comment by davidfischer** [comment]
Closing in favor of https://github.com/readthedocs/ethical-ad-server/pull/895 and https://github.com/readthedocs/ethical-ad-server/pull/896

**Pull request closed without merging**